pkg/feature/api/feature.go

```diff
@@ -2106,3 +2215,27 @@ func (s *FeatureService) CloneFeature(
 	}
 	return &featureproto.CloneFeatureResponse{}, nil
 }
+
+func (s *FeatureService) refreshFeaturesCache(ctx context.Context, environmentNamespace string) error {
+	fs, _, _, err := s.listFeatures(
+		ctx,
+		mysql.QueryNoLimit,
+		"",
+		nil,
+		"",
+		nil,
+		wrapperspb.Bool(false),
```

Comment:
To have the same behavior as existing implementations, I added the `wrapperspb.Bool(false)` temporarily to filter out  archived features from the Redis.

When we adopt the new evaluation method, we must set this arg to `nil` in order to include archived features to response.